src/gretel_trainer/relational/tasks/common.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+
+from gretel_client.projects.projects import Project
+from typing_extensions import Protocol
+
+from gretel_trainer.relational.core import RelationalData
+from gretel_trainer.relational.strategies.ancestral import AncestralStrategy
+from gretel_trainer.relational.strategies.independent import IndependentStrategy
+
+
+class _MultiTable(Protocol):
+    @property
+    def _refresh_interval(self) -> int:
+        ...
+
+    @property
+    def _project(self) -> Project:
```

Comment:
Oooo does this mean we could add project name/link to relational report?
(Don't worry, I'm not trying to do this now)

Reply:
Yep that's definitely available if we want it on the report.